src/components/ConfirmModal/index.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import {
+    TouchableOpacity, Text, View,
+} from 'react-native';
+import PropTypes from 'prop-types';
+import Header from '../Header';
+import Modal from '../Modal';
+import styles from '../../styles/styles';
+import CONST from '../../CONST';
+import withWindowDimensions from '../withWindowDimensions';
+
+const propTypes = {
+    /** Title of the modal */
+    title: PropTypes.string.isRequired,
+
+    /** A callback to call when the form has been submitted */
+    onConfirm: PropTypes.func.isRequired,
+
+    /** A callback to call when the form has been closed */
+    onCancel: PropTypes.func.isRequired,
+
+    /** If the screen is small */
+    isSmallScreenWidth: PropTypes.bool.isRequired,
+
+    /** Modal visibility */
+    isVisible: PropTypes.bool.isRequired,
+
+    /** Confirm button text */
+    confirmText: PropTypes.string,
+
+    /** Cancel button text */
+    cancelText: PropTypes.string,
+
+    /** Modal content text */
+    prompt: PropTypes.string,
+};
+
+const defaultProps = {
+    confirmText: 'Yes',
+    cancelText: 'No',
+    prompt: '',
+};
+
+const ConfirmModal = ({
+    title, onConfirm, onCancel, confirmText, cancelText, prompt, isSmallScreenWidth, isVisible,
+}) => {
+    const modalType = isSmallScreenWidth
+        ? CONST.MODAL.MODAL_TYPE.BOTTOM_DOCKED : CONST.MODAL.MODAL_TYPE.CONFIRM;
```

Comment:
Same here as before with the multiline ternary style

Reply:
Actually, I think you should just get rid of this local variable and use this same ternary directly when you pass the `type` prop to the `Modal` component within.

Reply:
Sure. will move ternary to type prop. Then we'll be able to remove `return` as well.